packages/cli/templates/react/es6/projects/empty/files/src/igniteuiResources.js

```diff
@@ -0,0 +1,7 @@
+// Ignite UI Required Style files
+export * from "ignite-ui/css/themes/infragistics/infragistics.theme.css";
```

Comment:
```suggestion
import "ignite-ui/css/themes/infragistics/infragistics.theme.css";
```

Even with this fix I get the following error, when running the project:

![image](https://user-images.githubusercontent.com/2621802/135458492-590ef934-be18-4383-bb11-fd689ea2cabd.png)

Reply:
I was also able to reproduce the same issue.